recipe/meta.yaml

```diff
@@ -1,50 +1,46 @@
-{% set version = "6.0.1" %}
-{% set sha256 = "6b8573841f2f7b60ffab9715c55dceff4f2a44e5a6d590ac189d20e8e7472714" %}
+{% set version = "10.0.0" %}
+{% set sha256 = "dd1ffcb42ed033f5167089ec4c6ebe84fbca1db4a9eaebf5c614af09d89eb135" %}
 
 package:
   name: lldb
   version: {{ version }}
 
 source:
-  url: http://releases.llvm.org/{{ version }}/lldb-{{ version }}.src.tar.xz
+  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version }}/lldb-{{ version }}.src.tar.xz
   sha256: {{ sha256 }}
-  patches:
-    # Remove use of private header in LLDB
-    # ref: https://reviews.llvm.org/D41824
-    - D41824.patch
 
 build:
   number: 0
-  skip: true  # [win and py<35]
-  features:
-    - vc14    # [win and py>=35]
+  # Windows and python 3.7 not compatible
```

Comment:
I'm not sure what you mean here. Can you explain?

Reply:
<img width="395" alt="Screen Shot 2020-04-09 at 1 53 46 PM" src="https://user-images.githubusercontent.com/1186124/78925504-89c89480-7a69-11ea-804f-d7c4b5dfd6b7.png">
https://lldb.llvm.org/resources/build.html#windows

Reply:
But it looks like my command didn't work and the windows 3.7 build is now passing? So I guess I can just get rid of the comment and the directive?